src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/49.diff.sql

```diff
@@ -0,0 +1,587 @@
+   ,Text2                     nvarchar(256) COLLATE Latin1_General_100_CI_AI_SC NOT NULL
+   ,TextOverflow2             nvarchar(max) COLLATE Latin1_General_100_CI_AI_SC NULL
+)
+GO
+IF NOT EXISTS (SELECT * FROM sys.types WHERE name = 'TokenSearchParamList')
+CREATE TYPE dbo.TokenSearchParamList AS TABLE
+(
+    ResourceTypeId           smallint NOT NULL
+   ,ResourceSurrogateId      bigint   NOT NULL
+   ,SearchParamId            smallint NOT NULL
+   ,SystemId                 int      NULL
+   ,Code                     varchar(256) COLLATE Latin1_General_100_CS_AS NOT NULL
+   ,CodeOverflow             varchar(max) COLLATE Latin1_General_100_CS_AS NULL
+)
+GO
+IF NOT EXISTS (SELECT * FROM sys.types WHERE name = 'TokenQuantityCompositeSearchParamList')
+CREATE TYPE dbo.TokenQuantityCompositeSearchParamList AS TABLE
+(
+    ResourceTypeId           smallint NOT NULL
+   ,ResourceSurrogateId      bigint   NOT NULL
+   ,SearchParamId            smallint NOT NULL
+   ,SystemId1                int      NULL
+   ,Code1                    varchar(256) COLLATE Latin1_General_100_CS_AS NOT NULL
+   ,CodeOverflow1            varchar(max) COLLATE Latin1_General_100_CS_AS NULL
+   ,SystemId2                int      NULL
+   ,QuantityCodeId2          int      NULL
+   ,SingleValue2             decimal(18,6) NULL
+   ,LowValue2                decimal(18,6) NULL
+   ,HighValue2               decimal(18,6) NULL
+)
+GO
+IF NOT EXISTS (SELECT * FROM sys.types WHERE name = 'TokenNumberNumberCompositeSearchParamList')
+CREATE TYPE dbo.TokenNumberNumberCompositeSearchParamList AS TABLE
+(
+    ResourceTypeId           smallint NOT NULL
+   ,ResourceSurrogateId      bigint   NOT NULL
+   ,SearchParamId            smallint NOT NULL
+   ,SystemId1 int NULL
+   ,Code1 varchar(256) COLLATE Latin1_General_100_CS_AS NOT NULL
+   ,CodeOverflow1 varchar(max) COLLATE Latin1_General_100_CS_AS NULL
+   ,SingleValue2 decimal(18,6) NULL
+   ,LowValue2 decimal(18,6) NULL
+   ,HighValue2 decimal(18,6) NULL
+   ,SingleValue3 decimal(18,6) NULL
+   ,LowValue3 decimal(18,6) NULL
+   ,HighValue3 decimal(18,6) NULL
+   ,HasRange bit NOT NULL
+)
+GO
+IF NOT EXISTS (SELECT * FROM sys.types WHERE name = 'TokenDateTimeCompositeSearchParamList')
+CREATE TYPE dbo.TokenDateTimeCompositeSearchParamList AS TABLE
+(
+    ResourceTypeId            smallint NOT NULL
+   ,ResourceSurrogateId       bigint   NOT NULL
+   ,SearchParamId             smallint NOT NULL
+   ,SystemId1                 int      NULL
+   ,Code1                     varchar(256) COLLATE Latin1_General_100_CS_AS NOT NULL
+   ,CodeOverflow1             varchar(max) COLLATE Latin1_General_100_CS_AS NULL
+   ,StartDateTime2            datetimeoffset(7) NOT NULL
+   ,EndDateTime2              datetimeoffset(7) NOT NULL
+   ,IsLongerThanADay2         bit      NOT NULL
+)
+GO
+IF NOT EXISTS (SELECT * FROM sys.types WHERE name = 'StringSearchParamList')
+CREATE TYPE dbo.StringSearchParamList AS TABLE
```

Comment:
I've noticed that some of these types don't have primary keys, should all they have it?